src/ui-components/VotesProgressInListing.tsx

```
@@ -164,23 +164,30 @@ const VotesProgressInListing = ({ tally, index, onchainId, status, proposalType,
-				if (parsedReferendumInfo?.ongoing?.tally) {
-					setTallyData({
-						ayes:
-							typeof parsedReferendumInfo.ongoing.tally.ayes === 'string'
-								? new BN(parsedReferendumInfo.ongoing.tally.ayes.slice(2), 'hex')
-								: new BN(parsedReferendumInfo.ongoing.tally.ayes),
-						nays:
-							typeof parsedReferendumInfo.ongoing.tally.nays === 'string'
-								? new BN(parsedReferendumInfo.ongoing.tally.nays.slice(2), 'hex')
-								: new BN(parsedReferendumInfo.ongoing.tally.nays)
-					});
+				if (api?.query?.referenda && api?.query?.referenda?.referendumInfoFor) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)